### PR TITLE
ROC-5421 - Defendant phone number by defendant

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5421

### Change description ###
Defendant phone number by defendant

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartyDetailsContentProvider.java [hunk 1]
```diff
@@ -68,6 +69,9 @@ public PartyDetailsContent     createContent(
 
         boolean nameAmended = !oppositeParty.getName().equals(ownParty.getName());
         boolean addressAmended = !oppositeParty.getAddress().equals(ownParty.getAddress());
+        boolean phoneAmended = oppositeParty.getPhone().isPresent()
+            && !(oppositeParty.getPhone().orElse("")
+            .equalsIgnoreCase(ownParty.getMobilePhone().orElse("")));
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartyDetailsContentProvider.java]: The `isPresent()` check isn't needed

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java [hunk 2]
```diff
@@ -82,6 +85,10 @@ public boolean getAddressAmended() {
         return addressAmended;
     }
 
+    public boolean getPhoneAmended() {
+        return phoneAmended;
+    }
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: Convention is to use `is` as the prefix for a boolean getter

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartyDetailsContentProvider.java (hunk 1 above)
Comment (karthikkannankainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartyDetailsContentProvider.java]: its needed, as we check if it was provided by opposite party and then changed by self

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java (hunk 2 above)
Comment (karthikkannankainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: the template engine looks for `get`  when I did `isPhoneAmended` it failed. So defaulted to `get`

Comment (andrewwa-kainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: Okey dokey!

Comment (karthikkannankainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: thanks @andrewwa-kainos

**Pull request merged**